tests/checks/scaling_phase_statefulset/scaling_phase_statefulset_test.go

```diff
@@ -100,7 +100,7 @@ spec:
     spec:
       containers:
       - name: apache-ab
-        image: ghcr.io/kedacore/tests-apache-ab
```

Comment:
just trying this before official builds are ready
https://github.com/kedacore/test-tools/pull/229